src/graphorm/index.ts

```diff
@@ -1715,6 +1716,183 @@ const obj = new GraphORM({
       },
     },
   },
+  OpportunityPreviewUser: {
+    from: 'User',
+    requiredColumns: ['id'],
+    fields: {
+      profileImage: {
+        select: 'image',
+      },
+      anonId: {
+        // Will be set by transform using parent data
+        select: () => 'NULL',
```

Comment:
should we just overwrite `id`? Or at least hide it so then somebody does not take it from response and gets the real user?